mnelab/mainwindow.py

```diff
@@ -222,6 +222,17 @@ def __init__(self, model):
         self.actions["events_from_annotations"] = tools_menu.addAction(
             "Create events from annotations", self.events_from_annotations
         )
+        tools_menu.addSeparator()
+        self.export_menu = tools_menu.addMenu("NIRS")
+        self.actions["convert_od"] = self.export_menu.addAction(
+                "Convert &to optical density", self.convert_od)
```

Comment:
We need to have unique mnemonics, i.e. &t is used by all three menu entries here. I'd put them in front of some meaningful letter, e.g. &density here.